action.yml

```diff
@@ -34,8 +34,15 @@ runs:
       id: action
       shell: powershell
       run: |
-        $service_credential_password = ConvertTo-SecureString -String '${{ inputs.service-credential-password }}' -AsPlainText -Force
-        $secure_password = ConvertTo-SecureString -String '${{ inputs.service-account-password }}' -AsPlainText -Force
+        $service_password_string = @'
+        ${{ inputs.service-credential-password }}
+        '@
+        $service_credential_password = ConvertTo-SecureString -String $service_password_string -AsPlainText -Force
+
+        $deployment_password_string = @'
+        ${{ inputs.service-account-password }}
+        '@
+        $secure_password = ConvertTo-SecureString -String $deployment_password_string -AsPlainText -Force
```

Comment:
Is there something wrong with my example that used the environment variable? Its fewer lines of code and, I think, easier to read.

Reply:
It is much easier to read, but when I tested it (https://github.com/im-open/deploy-windows-files/pull/9/commits/19a2779d828bc15cab4130c5faefbb56679ce4b1) I got the same errors.